Frontend/ui-customer/src/Helpers/AppVersionHelper.purs

```diff
@@ -0,0 +1,95 @@
+module Helpers.AppVersionHelper where
+
+import Prelude
+import Types.App (FlowBT, ScreenType(..))
+import JBridge as JB
+import Presto.Core.Types.Language.Flow (getLogFields)
+import MerchantConfig.Utils (Merchant(..), getMerchant)
+import Screens.AppUpdatePopUp.Handler as UI
+import Control.Monad.Except.Trans (lift)
+import Engineering.Helpers.Commons (liftFlow, os)
+import Engineering.Helpers.LogEvent (logEvent)
+import Foreign (unsafeToForeign)
+import Common.Types.App (LazyCheck(..))
+import Data.Int as INT
+import Data.Maybe (fromMaybe)
+import Data.String (split, Pattern(..))
+import Engineering.Helpers.BackTrack (liftFlowBT)
+import ModifyScreenState
+import Screens.Types (UpdatePopupType(..))
+import Data.Array ((!!), any)
+
+type IosVersion = {
+  majorUpdateIndex :: Int,
+  minorUpdateIndex :: Int,
+  patchUpdateIndex :: Int,
+  enableForceUpdateIOS :: Boolean
+}
+
+checkVersion :: FlowBT String Unit
+checkVersion = do
+  versionCodeAndroid <- lift $ lift $ liftFlow $ JB.getVersionCode
+  versionName <- lift $ lift $ liftFlow $ JB.getVersionName
+  void $ pure $ JB.setCleverTapUserProp [ {key : "Platform", value : unsafeToForeign os}]
+  logField_ <- lift $ lift $ getLogFields
+  let updatedIOSversion = getIosVersion (getMerchant FunctionCall)
+  if os /= "IOS" && versionCodeAndroid < (getLatestAndroidVersion (getMerchant FunctionCall)) then do
```

Comment:
let's see if this format of if else is more readable

``` purescript
if cond
  then do
      ....
      ...
  else
```

Reply:
Yeah, actually I just moved this file. Haven't modified the content of it yet. Will refactor those as well 👍